src/Post/Comments/Comment.js

```diff
@@ -5,15 +5,17 @@ import ListItemAvatar from '@material-ui/core/ListItemAvatar';
 import Avatar from '@material-ui/core/Avatar';
 
 const Comment = ({ user, comment, date }) => (
-    <ListItem>
-        <ListItemAvatar>
-            <Avatar alt={user.name} src={user.avatar} />
-        </ListItemAvatar>
-        <ListItemText
-            primary={comment}
-            secondary={date.toDateString()}
-        />
-    </ListItem>
+    <>
```

Comment:
@opmanual  you no longer need this outer tag since "ListItem" is the only root-level tag.